module-1/views/contacts-admin.ejs

```diff
@@ -69,16 +88,9 @@
 						subject: `re: ${message}`,
 						text: text.value 
 					}
-					fetch('http://localhost:3001/sendEmail', {
-						method: 'POST',
-						headers: {
-							'Content-Type': 'application/json',
-						},
-						body: JSON.stringify(data),
-					})
-					.then(response => response.json())
+					postData('http://localhost:3001/sendEmail', data)
```

Comment:
Instead of putting a full url string in every request create a separate file with constants and export an APP_URL constant from that file and import it here and use like postData(`${APP_URL}/sendEmail`)"